docs/recipes/reasonml.md

```diff
@@ -0,0 +1,43 @@
+# [ReasonML](https://reasonml.github.io) / [BuckleScript](https://bucklescript.github.io)
+
+## Installation
+
+```sh
+npm install --save-dev ava
+```
+
+Then add `ava` to `bs-dev-dependencies` in your `bsconfig.json`:
+```js
+{
+  ...
+  "bs-dev-dependencies": ["ava"]
+}
+```
+
+## Getting started
+
+```ml
+open Sync;
+
+test("Should pass", t => {
+  t.deepEqual(1, 1);
+  t.pass();
+});
+```
+
+```ml
+open Async;
+
+test("Should pass", t => {
+  Js.Global.setTimeout(() => t.cb(), 1000)
+  |> ignore
+});
+```
+
+```ml
+open Promise;
+
+test("Should pass", t => {
+  t.notThrows(Js.Promise.resolve(true"));
```

Comment:
This should be `await t.notThrowsAsync(Js.Promise.resolve(true"));`.

Reply:
ReasonML returns last statement of function. It's like following in javascript

```js
test("function returns a fulfilled promise", t => {
  return t.notThrowsAsync(() => Promise.resolve(true));
});
````